package.json

```diff
@@ -79,6 +80,8 @@
     "@types/node": "^16.10.3",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
+    "chai": "^4.3.4",
+    "codeclimate": "^0.0.32",
```

Comment:
**praises:** nice catch here !

It was not added because it was using my codeclimate global instance locally !
Which is not the case for everyone.
Thanks for adding it

Reply:
Yep, that's why I included it ... it wasn't found on my system :D